Source/OxyPlot/PlotModel/PlotModel.cs

```diff
@@ -193,6 +193,16 @@ public enum TitleHorizontalAlignment
     public partial class PlotModel : Model, IPlotModel
     {
         /// <summary>
+        /// The snap timer.
+        /// </summary>
+        private readonly Timer snapTimer;
+
+        /// <summary>
+        /// Gets or sets whether the axis is currently snapping.
+        /// </summary>
+        private bool isSnapping;
```

Comment:
This is required. Snapping might be enabled, but the axis might not be snapping *at the moment*. We need to prevent that because of the snapping "move", we don't spawn another snapping call (and cause unnecessary calls).